src/hist/plot.py

```diff
@@ -608,7 +615,7 @@ def _plot_ratiolike(
         if fit_fmt is not None:
             parnames = list(inspect.signature(other).parameters)[1:]
             popt, pcov = bestfit_result
-            perr = np.sqrt(np.diag(pcov))
+            perr = np.sqrt(np.diag(pcov))  # type: ignore
```

Comment:
Can we switch to using `np.diagonal`? `np.diag` is a Matlab compat function, which is why it's untyped, I think.

Reply:
sure, let me check.